### DISCUSSION TITLE
What orange circle mark near plugin means in lazy pane?

### DISCUSSION THREAD

**Discussion by Demianeen** [LazyVim/LazyVim#3568] [Q&A] [answered]
I had added a new plugin and expected lazy to download it, but I just get this orange mark:
![CleanShot 2024-06-10 at 13 25 53@2x](https://github.com/LazyVim/LazyVim/assets/51330172/3ca29325-9e57-43ab-9374-dd501d7e437a)

When I try to require the plugin I get error that the luarock 'op' is not found. I thought to delete it, but normal x does nothing:

https://github.com/LazyVim/LazyVim/assets/51330172/71650804-3b8e-4386-a656-d57d6e9c0319

I also can't find the plugin folder in `~/.local/share/nvim`

What should I do in this case?

**Comment by folke** [comment, marked as answer]
That means it's a dev plugin. Check the lazy.nvim docs and your config.
Edit: they're orange on Tokyonight, but you dont have tokyonight it seems
